packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@web3-onboard/core",
-  "version": "2.20.5-alpha.1",
+  "version": "2.20.5-alpha.2",
```

Comment:
```suggestion
  "version": "2.20.5-alpha.1",
```

Reply:
We aren't adding native chain support for more chains at this time as it is easily possible for devs to add chains within their init. This is because we are trying to keep our package ultra slim and only deliver to devs what they need and allow for configuration on their end.
If you are part of the Ronin team please reach out to mobeen@blocknative.com to speak further on support within the BN ecosystem